test/test_rosbag/test/test_bag.py

```diff
@@ -188,6 +190,29 @@ def test_rosbag_filter(self):
 
         self.assertEquals(len(msgs), 5)
 
+    def test_trivial_rosbag_filter(self):
+        # regression test for issue #736
+
+        inbag_filename  = '/tmp/test_rosbag_filter__1.bag'
+        outbag1_filename = '/tmp/test_rosbag_filter__2.bag'
+        outbag2_filename = '/tmp/test_rosbag_filter__3.bag'
```

Comment:
Please avoid hard coding temporary paths. Use `tempfile` instead. That will ensure that the names don't collide with other tests and potentially work on other platforms.